README.md

```diff
@@ -181,6 +181,7 @@ config:
   list_objects_version: ""
   bucket_lookup_type: auto
   send_content_md5: true
+  disable_multipart: false
```

Comment:
A bit unfortunate that we have to use a double negation, but I don't see another way to preserve backwards compatibility.